ClientApp/viking-vault/src/components/SideBar.tsx

```diff
@@ -31,28 +33,37 @@ class SideBar extends React.Component<any, ISideBarState> {
     }
 
     render(){
+        
+        let sidebarVisibility:string = this.state.show ? "sidebar collapsed" : "sidebar expanded";
```

Comment:
Best practice in JavaScript is that if that variable is not changed, you make it a `const`